src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -89,7 +89,10 @@ Tap here to learn more about DeArrow."</string>
     <string name="revanced_alt_thumbnail_stills_time_entry_3">End of video</string>
     <string name="revanced_alt_thumbnail_dearrow_error">DeArrow temporarily unavailable. (status code: %s)</string>
     <string name="revanced_alt_thumbnail_dearrow_error_generic">DeArrow temporarily unavailable.</string>
-
+    <string name="revanced_alt_thumbnail_use_alternative_domain_title">Use alternative domain for images</string>
+    <string name="revanced_alt_thumbnail_use_alternative_domain_summary">Replaces the domain that is blocked in some regions so that video thumbnails, channel avatars, community post images, etc. can be received.</string>
+    <string name="revanced_alt_thumbnail_alternative_domain_title">Alternative domain</string>
+    <string name="revanced_alt_thumbnail_alternative_domain_summary">The domain to fetch images from.\nNote: Only enter the domain name, without an \"https\:\/\/\" prefix.</string>
```

Comment:
```suggestion
    <string name="revanced_alt_thumbnail_alternative_domain_summary">The domain to fetch images from.\nNote: Only enter the domain name, i.e., without the \"https\:\/\/\" prefix.</string>
```

Like this?

Reply:
if @KobeW50  approves that, then I'll change. English not my strong side